File: frontend/eBolnica/src/app/models/medical-record.dto.ts
export interface MedicalRecord{
    id: number, 
    patientId: number,
    recordNumber: string
    firstName: string;
    lastName: string;
    dateOfBirth?: Date;
    gender?: string;
    phoneNumber?: string;
    address?: string;
    bloodType?: string;
    isAdmitted?: boolean;
    email:string;
    reports: MedicalReport[];
}

export interface MedicalReport{
  doctorId: number;
  createdAt: string;
  diagnosis: string;
  therapy: string;
  symptoms: string;
}

export interface newMedicalReport{
    medicalRecordId: number,
    description: string,
    diagnosis: string,
    therapy: string,
    symptoms: string
}

File: backend/eBolnicaAPI/eBolnicaAPI/Models/DTOs/MedicalRecordDto.cs

namespace eBolnicaAPI.Models.DTOs
{
    public class MedicalRecordDto
    {
        public int Id { get; set; }

        public int PatientId { get; set; }

        public string RecordNumber { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public DateTime? DateOfBirth { get; set; }

        public string Gender { get; set; }

        public string PhoneNumber { get; set; }

        public string Address { get; set; }

        public string BloodType { get; set; }

        public bool? IsAdmitted { get; set; }

        public string Email { get; set; }
        public List<MedicalReportCreateDto> Reports { get; internal set; }
    }
}
